fake_joint_driver/include/fake_joint_driver/fake_joint_driver.h

```diff
@@ -11,14 +11,15 @@
 #include <hardware_interface/joint_state_interface.h>
 #include <hardware_interface/robot_hw.h>
 
-class FakeJointDriver : public hardware_interface::RobotHW
-{
+class FakeJointDriver : public hardware_interface::RobotHW {
 private:
   hardware_interface::JointStateInterface joint_state_interface;
   hardware_interface::PositionJointInterface position_joint_interface;
   hardware_interface::VelocityJointInterface velocity_joint_interface;
 
   std::vector<double> cmd_dis;
```

Comment:
Note:  class members should be named with a trailing underscore. Like `joint_state_interface_`

I know this wasn't your mistake initially, you're just copying the existing format. I think it's worthwhile to make this change b/c it would help with readability a lot!

Reply:
https://github.com/PickNikRobotics/fake_joint/pull/1/commits/a1941184a48cb560eab2f970460d4cb4d587a669